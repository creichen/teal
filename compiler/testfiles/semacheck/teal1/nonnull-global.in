var x : nonnull int;

fun main() = {
   x := 5;
   return x;
}