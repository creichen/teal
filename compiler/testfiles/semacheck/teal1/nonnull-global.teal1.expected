testfiles/semacheck/teal1/nonnull-global.in[1:0-1:19]: Global variable types cannot be qualified.
